routers/web/feed/profile.go

```diff
@@ -34,6 +34,9 @@ func RetrieveFeeds(ctx *context.Context, options models.GetFeedsOptions) []*mode
 	}
 
 	for _, act := range actions {
+		if act.Repo == nil {
+			continue
+		}
```

Comment:
This could just push the NPE further downstream if the downstream function still assumes that the repo is extant.

Have we checked the downstream functions here? Would it make more sense to censor and remove actions that have non-extant repos?

Reply:
Hmm it looks like this should be fine as these cases should not occur.